src/pocketmine/entity/Human.php

```diff
@@ -546,6 +571,7 @@ public function close(){
 				}
 
 				$this->inventory = null;
+				$this->enderChestInventory = null;
```

Comment:
you aren't removing it from viewers, which might cause leaks if plugins open it to other players.